spec/cypress/end_to_end/create_wellplate.cy.js

```diff
@@ -0,0 +1,35 @@
+const { element } = require('prop-types');
```

Comment:
`element` appears to be unused?

Reply:
Some (unwanted) code autogenerated by cypress. I missed it to delete while pushing changes. So now deleted again.